src/js/samples/components/Files/SampleReads.tsx

```diff
@@ -29,15 +35,3 @@ function SampleReads({ reads, prefix }) {
         </BoxGroup>
     );
 }
-
-function mapStateToProps(state) {
-    const { id, reads, name } = state.samples.detail;
-
-    return {
-        reads,
-        id,
-        prefix: snakeCase(name),
```

Comment:
Was prefix never used before?

Reply:
No I believe it originally got passed to `ReadItem` but was never actually used